src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+export type Tab = {
+  id: string
+  title: string
+  content: string
+};
+
+type TabsProps = {
+  tabs: Tab[]
+  selectedTabId: string
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  let selectedTabIdCopy = selectedTabId;
+
+  if (tabs.length > 0 && !tabs.map(tab => tab.id).includes(selectedTabId)) {
+    selectedTabIdCopy = tabs[0].id;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = tab.id === selectedTabIdCopy;
+
+            return (
+              <li
+                data-cy="Tab"
+                key={tab.id}
+                className={isTabSelected ? 'is-active' : ''}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={
```

Comment:
create separate logic for this click to be like, e.g.

```
const handleTabSelect = (tab: Tab) => {
    if (tab.id !== selectedTabId) {
      onTabSelected(tab);
    }
  };
```

and use here like:

`onClick={() => handleTabSelect(tab)}`

it will be easier and more readable

Reply:
I created function.